docs/asciidoc/modules/ROOT/pages/export/arrow.adoc

```diff
@@ -0,0 +1,104 @@
+[[export-arrow]]
+= Export to Apache Arrow
+:description: This section describes procedures that can be used to export data in Apache Arrow format.
+
+The export Apache Arrow procedures export data into a format that's used by many Apache tools and non.
```

Comment:
These procedures export data into a format that's used by many Apache and non-Apache tools.